docs/src/_cloud/onboarding.md

```diff
@@ -117,7 +106,7 @@ meltano-cloud history
 meltano-cloud schedule list
 ```
 
-### Step 5: Initialize secrets
```

Comment:
It might be helpful to mention that secrets are what are normally added to your .env file automatically by Meltano during configuration. Might be obvious for some but not for others.

Reply:
+1 to this. It came up in a user call today as well.